mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -2570,12 +2581,20 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportDeletes(TALLOC_CTX *mem_ctx,
 	owner = emsmdbp_get_owner(synccontext_object);
 	openchangedb_get_MailboxReplica(emsmdbp_ctx->oc_ctx, owner, &repl_id, &replica_guid);
```

Comment:
I see you have added retval checks on openchangedb_get_MailboxReplica in another part of the patch, please propagate this change here too.